idl/github.com/uber/cadence/indexer.thrift

```diff
@@ -50,5 +48,5 @@ struct Message {
   30: optional string workflowID
   40: optional string runID
   50: optional i64 (js.type = "Long") version
-  60: optional IndexAttributes indexAttributes
+  60: optional map<string,Field> fields
```

Comment:
I thought we are planning to create two separate abstractions Indexed Fields and Non-Indexed Fields.

Reply:
Later I realize there is no need to have separate abstractions for Indexed and Non-Indexed Fields. The control of index or not is on ES sever, where we put schema in index.